Rakefile

```diff
@@ -1,22 +1,77 @@
 require 'bundler'
+Bundler.require(:rake)
 
 # devel
 require 'puppet_blacksmith/rake_tasks' if Bundler.rubygems.find_name('puppet-blacksmith').any?
+require 'puppetlabs_spec_helper/rake_tasks' if Bundler.rubygems.find_name('puppetlabs_spec_helper').any?
+if Bundler.rubygems.find_name('puppet-lint').any?
+  require 'puppet-lint/tasks/puppet-lint'
+  PuppetLint.configuration.ignore_paths = ["spec/**/*.pp", "vendor/**/*.pp"]
+  PuppetLint.configuration.log_format = '%{path}:%{linenumber}:%{KIND}: %{message}'
+  PuppetLint.configuration.send("disable_class_inherits_from_params_class")
+  PuppetLint.configuration.send("disable_80chars")
+end
 
 # debug
 require 'pry' if Bundler.rubygems.find_name('pry').any?
 
-Bundler.require(:rake)
+# build
+if Bundler.rubygems.find_name('mof').any?
+  Dir.glob('build/*.rb').each { |r| import r }
+  Dir.glob('build/*.rake').each { |r| import r }
+end
+
+# tasks
+desc 'Run the Acceptance Tests'
+task :acceptance_tests, :tests, :config, :puppet_agent_ver do |t, args|
```

Comment:
@ferventcoder This was a bit of experimentation just to see how we may want to call tests from rake tasks in the future. Before we formalize on a rake task testing strategy the test tiering RFC needs to be finalized. (Dom is working on that currently.)